apiController.js

```diff
@@ -120,3 +124,19 @@ exports.patchVotes = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comments_id } = req.params;
+  const commentIdInt = parseInt(comments_id)
+
+return commentIdExists(commentIdInt)
```

Comment:
You don't need to check that the comment exists - if you try to delete a comment and psql cant find it - you will get an error from psql.